integrationtests/tests/fixtures/mod.rs

```diff
@@ -117,8 +117,7 @@ pub async fn fixtures(
     match env::var("MINIMINT_TEST_REAL") {
         Ok(s) if s == "1" => {
             info!("Testing with REAL Bitcoin and Lightning services");
-            let dir =
-                env::var("MINIMINT_TEST_DIR").expect("Must have test dir defined for real tests");
+            let dir = env::var("FM_TEST_DIR").expect("Must have test dir defined for real tests");
```

Comment:
I thought about calling it `FM_MINIMINT_TEST_DIR`, but that seemed overly verbose. Also wanted to avoid calling it `MM_*`, then we'd just touch it again soon.